src/main/java/ca/on/oicr/pineryreports/reports/impl/LanesBillingReport.java

```diff
@@ -48,7 +48,8 @@
     private final String runStatus;
     private final String sequencingParameters;
     
-    public DetailedObject(RunDto run, String instrumentName, String instrumentModel, String laneNumber, String project,
+    public DetailedObject(RunDto run, String instrumentName, String instrumentModel, int novaSeqLanesCount, String laneNumber,
```

Comment:
document that `novaSeqLanesCount` should be 0 for non-NovaSeq runs

Reply:
Done